UnitTests/UnitTests.UWP/UI/Controls/Test_ListDetailsView_UI.cs

```diff
@@ -0,0 +1,113 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Uwp;
+using Microsoft.Toolkit.Uwp.UI;
+using Microsoft.Toolkit.Uwp.UI.Controls;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Markup;
+
+namespace UnitTests.UWP.UI.Controls
+{
+    [TestClass]
+    public class Test_ListDetailsView_UI : VisualUITestBase
+    {
+        private const string sampleXaml = @"<controls:ListDetailsView
```

Comment:
StyleCop: `D:\a\1\s\UnitTests\UnitTests.UWP\UI\Controls\Test_ListDetailsView_UI.cs(21,30): error SA1303: Const field names should begin with upper-case letter. [D:\a\1\s\UnitTests\UnitTests.UWP\UnitTests.UWP.csproj]`